WSL/wsl-config.md

```diff
@@ -291,3 +291,37 @@ These options are available in Insider Build 17713 and later.
 |:----|:----|:----|:----|
 | enabled | boolean | `true` | Setting this key will determine whether WSL will support launching Windows processes. |
 | appendWindowsPath | boolean | `true` | Setting this key will determine whether WSL will add Windows path elements to the $PATH environment variable. |
+
+#### user
+
+Section label: `[user]`
+
+These options are available in Build 18980 and later.
+
+| key | value | default | notes|
+|:----|:----|:----|:----|
+| default | string | The initial username created on first run | Setting this key specifies which user to run as when first starting a WSL session. |
+
+## Configure global options with .wslconfig
+
+> **Available in Windows Build 19041 and later**
+
+You can configure global WSL options by placing a `.wslconfig` file into the root directory of your users folder: `C:\Users\<yourUserName>\.wslconfig`. This file can contain the following options:
+
+### WSL 2 Settings
+
+These settings affect the VM that powers any WSL 2 distribution. 
+
+| key | value | default | notes|
+|:----|:----|:----|:----|
+| kernel | string | The Microsoft built kernel provided inbox | An absolute Windows path to a custom Linux kernel. |
+| memory | size | 70% of your total memory on Windows | How much memory to assign to the WSL 2 VM. |
+| processors | number | The same number of processors on Windows | How many processors to assign ot the WSL 2 VM. |
+| localhostForwarding | boolean | `true` | Boolean specifying if ports bound to wildcard or localhost in the WSL 2 VM should be connectable from the host via localhost:port. |
+| kernelCommandLine | string | Blank | Additional kernel command line arguments. |
+| swap | size | ??? 30GB? | How much swap space to add to the WSL 2 VM, 0 for no swap file. |
+| swapFile | size | Same directory as your distro??? | An absolute Windows path to the swap virtual hard disk. |
```

Comment:
@benhillis and last one here :)

Reply:
%USERPROFILE%\AppData\Local\Temp\swap.vhdx